src/main/java/com/google/api/codegen/config/ResourceNameMessageConfigs.java

```diff
@@ -77,6 +78,7 @@ static ResourceNameMessageConfigs createFromAnnotations(
                   message,
                   resourceField);
           if (Strings.isNullOrEmpty(entityName)) continue;
+          if (!resourceNameConfigs.containsKey(entityName)) continue;
```

Comment:
Please always use braces after `if`s (I see you made it consistent with the preivous if statement, please update the above if statement as well).